resources/views/macros/form.twig

```diff
@@ -1,7 +1,12 @@
 {% macro input(name, label, type, required) %}
     <div class="form-group">
         {% if label %}
-            <label for="{{ name }}">{{ label }}</label>
+            <label for="{{ name }}">
+                {{ label }}
+                {% if required|default(false) %}
+                    <span class="text-info glyphicon glyphicon-warning-sign"></span>
```

Comment:
I don't think that we should use the warning sign here, a smaller asterisk would be a better choice as this makro will be used every where and some input fields will be obviously required like a search input, so eventually use another parameter like `show_required` and only append an asterisk to the name (as this should be common knowledge)?

And PS: You should have used your own `entry_required` macro ;)